CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 
 ### Added 🚀
 
+-   Allow color of buildings to be customizable in the ribbon bar and in the legend. Colors of edges are now customizable as well ([#1533](https://github.com/MaibornWolff/codecharta/issues/1533))
```

Comment:
What about the "selected" color and the folder marking colors from the context menu? :)
Does a user know what we mean with "ribbon bar"? Maybe we can add an self-explaining screenshot for that?

Reply:
![image](https://user-images.githubusercontent.com/3596742/103547861-3c1c7380-4ea5-11eb-8df2-541caf65b9df.png)
upload here, for referring it in changelog 😇

Reply:
The screenshot is a great idea and I added the above one 👍 

- For me "color of buildings" includes "selected" and should be more clear with the screenshot now - or do you have a better suggestion @ce-bo ? :)
- @ce-bo Do you mean the following screenshot with _"the folder marking colors from the context menu"_? I don't remember, if we wanted to update this color picking in this FR or in a new one - guess it makes sense to also do it in this PR (@BridgeAR?)
![image](https://user-images.githubusercontent.com/3596742/103549100-20b26800-4ea7-11eb-924d-3a65a48ace97.png)